conda/activate.py

```diff
@@ -498,6 +507,15 @@ def ensure_binary(value):
         return value
 
 
+def ensure_unicode(value):
+    try:
+        return value.decode('unicode_escape')
```

Comment:
I don't think `'unicode_escape'` is correct.
I'd rather assume `"utf-8"` or maybe you'd have to use `sys.getfilesystemencoding()`? (just guessing)

Reply:
I just copied this from https://github.com/conda/conda/blob/4.4.7/conda/common/compat.py#L178-L184.  But yeah, maybe actually what we want is https://github.com/conda/conda/blob/4.4.7/conda/common/compat.py#L187-L191

Reply:
Or maybe all I really want is [`text_type`](https://github.com/conda/conda/blob/4.4.7/conda/common/compat.py#L37)

Reply:
I guess `text_type()` could throw UnicodeDecodeError?  And in that case we should also decode with the filesystem encoding?  I always talk myself in circles with this stuff.

Reply:
P.S. @mbargull Can you email me directly? I can't find your contact info.  kfranz at anaconda com